### PR TITLE
Add travel fund request for OpenJS World 2022

### PR DESCRIPTION
**Pull request by christian-bromann**
This is a travel fund request for participating at this year's OpenJS World 2022. I am speaking there and plan to attend the Collaborator Summit.

As member of the program committee I report that we have discussed to use the CPC budget to fund the travel to avoid putting a burden on the conference budget (cc @rginn).

I'm asking for 1100 USD, which is an estimate of the cost of my flights (Berlin -> Austin -> Barcelona) and transits to and from airports. Costs for hotel will be covered by the conference budget though.

cc @openjs-foundation/cpc

### CODE REVIEW

**Comment by christian-bromann** [comment]
The Link Checker seems to check links based on `main` branch rather than my patched one. Also some of the `404` mentioned broken links are actually working. Not sure how to resolve this check.

File: TRAVEL_FUND/2022.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# 2021 Member Travel Fund Record
```
Comment (eemeli) [inline TRAVEL_FUND/2022.md:1]:
```suggestion
# 2022 Member Travel Fund Record
```

**Review by eemeli — APPROVED** [review]

**Comment by WaleedAshraf** [comment]
> Also some of the 404 mentioned broken links are actually working.
They are not working for me. It was already failing on the master branch. So, those should be fixed in separate PR. 👍

**Review by WaleedAshraf — APPROVED** [review]

**Comment by ljharb** [comment]
Looks like the link checker here is failing *because* this PR is changing file paths, meaning, it will pass once this is merged.

**Review by mhdawson — APPROVED** [review]
LGTM

**Review by ljharb — APPROVED** [review]

**Review by joesepi — APPROVED** [review]

**Review by dylans — APPROVED** [review]

**Pull request merged**